python/py-pytorch/Portfile

```diff
@@ -94,35 +94,27 @@ if {${name} ne ${subport}} {
         port:cctools \
         path:bin/cmake:cmake \
         path:bin/ninja:ninja \
-        port:py${python.version}-astunparse \
         port:py${python.version}-requests \
         port:py${python.version}-setuptools
 
     depends_lib-append \
         path:share/pkgconfig/eigen3.pc:eigen3 \
-        port:gmp \
-        port:mpfr \
-        path:lib/libopenblas.dylib:OpenBLAS \
-        path:lib/opencv4/libopencv_core.dylib:opencv4 \
-        port:zmq \
-        port:zstd \
-        port:onetbb \
-        port:google-glog \
         port:gflags \
-        port:leveldb \
-        port:lmdb \
+        port:google-glog \
         port:libomp \
+        port:protobuf3-cpp \
         port:py${python.version}-click \
-        port:py${python.version}-cffi \
         port:py${python.version}-future \
         port:py${python.version}-numpy \
         port:py${python.version}-pybind11 \
         port:py${python.version}-sympy \
         port:py${python.version}-typing_extensions \
-        port:py${python.version}-yaml
+        port:py${python.version}-yaml \
+        port:zstd
 
     depends_run-append \
-        port:py${python.version}-onnx
+        port:py${python.version}-onnx \
```

Comment:
having the dependency doesn't do anything unless you set `USE_SYSTEM_ONNX` and that fails